WordPress/Classes/Utility/Blogging Prompts/PromptRemindersScheduler.swift

```diff
@@ -282,12 +291,11 @@ private extension PromptRemindersScheduler {
     func addStaticNotifications(after afterDate: Date,
                                 with schedule: Schedule,
                                 time: Time,
-                                blog: Blog,
+                                siteID: Int,
                                 maxDays: Int = Constants.staticNotificationMaxDays) -> [String]? {
         guard case .weekdays(let weekdays) = schedule,
               maxDays > 0,
-              let maxDate = Calendar.current.date(byAdding: .day, value: maxDays, to: afterDate),
-              blog.dotComID != nil else {
+              let maxDate = Calendar.current.date(byAdding: .day, value: maxDays, to: afterDate) else {
```

Comment:
Not related to this PR. But I didn't realise that [`Calendar.current`](https://developer.apple.com/documentation/foundation/calendar/2293438-current)—the calendar that user picked in the device's System Settings—is used a lot in the app. In most cases, we want the Gregorian calendar, not the calendar the user likes their phone to display.

Reply:
Hmm, yeah, I think I didn't really think about it and just followed how it's done across the codebase. I'll consider fixing this during the refactor later. Thanks!